processing/src/main/java/org/apache/druid/query/spec/SpecificSegmentQueryRunner.java

```diff
@@ -74,7 +74,7 @@ public Sequence<T> run(final QueryPlus<T> input, final ResponseContext responseC
 
     final Thread currThread = setName ? Thread.currentThread() : null;
     final String currThreadName = setName ? currThread.getName() : null;
-    final String newName = setName ? query.getType() + "_" + query.getDataSource() + "_" + query.getIntervals() : null;
+    final String newName = setName ? "processing_" + query.getId() : null;
```

Comment:
Would be useful to include the table names too, so people can see what it is without having to look up the query ID somewhere. Maybe `"processing_" + query.getDataSource().getTableNames() + "_" + query.getId()`

Reply:
Based on conversation here : https://github.com/apache/druid/pull/15059#discussion_r1342331267
leaving it at queryID.

Reply:
eh, i find it useful to be able to at a glance know query type and table name when looking at flame graphs, im not sure you can possibly know the table name now with this change without some external reference, though i suppose you can still tell the type by look into the flame graph...